include/integratorxx/atomic_densities/SlaterAtomicShell.hpp

```diff
@@ -96,7 +96,8 @@ class SlaterTypeAtomicShell {
   double evaluate_density(const double *orbs, const int_container &occs) {
     double density = 0.0;
     for(size_t iorb = 0; iorb < occs.size(); iorb++) {
-      double orbital_density = std::pow(orbs[iorb], 2);
+      double orbital_density = std::abs(orbs[iorb]);
+      orbital_density *= orbital_density;
```

Comment:
doesn't this result in taking the square root?!

Reply:
For complex,I suppose it does! (for real, no, it's an assembly bit-flip). In this instance, I would not consider it performance critical and it drastically improves the readability of the code.

Reply:
Rewrote this code